app/ui-react/syndesis/src/modules/support/pages/SupportPage.tsx

```diff
@@ -0,0 +1,19 @@
+import { PageSection, SupportHeader } from '@syndesis/ui';
+import * as React from 'react';
+import { PageTitle } from '../../../shared';
+
+export function SupportPage() {
+  return (
+    <>
+      <PageTitle title="Support" />
+      <SupportHeader
+        i18nPageTitle={'Support'}
+        i18nDescription={`To obtain support, download diagnostic information through this page and open a request on the <a href="https://access.redhat.com/support/cases/#/case/new">Red Hat Customer portal</a>. If you have any issues please see the support <a href="https://access.redhat.com/solutions/2112">instructions</a>.`}
+      />
+
+      <PageSection>
+        <h2>TODO</h2>
```

Comment:
Currently, we show the same content you added to the about dialog plus a form to download some extra diagnostic. I think we should implement it as it is already since it's all info that can be handy when opening a ticket.

Reply:
@riccardo-forina I've added the basic version information and addressed the other bits of feedback. I think we're at a good point in time to merge what we have now and can flesh out the rest of the support page in a follow PR. I'd also like to work in the WithLoader support since there are api calls involved here, and don't want to drag this PR out for all that. Do you think what I've added so far enough for now?